tests/MavenTests/MavenTests.pro

```diff
@@ -28,7 +28,11 @@ LIBS += -lmaven -lpugixml -lneural -lcsvparser -lpls -lErrorHandling -lLogger -l
 !macx: LIBS += -fopenmp
 
 macx {
-LIBS -= -lnetcdf -lcdfread
+    QMAKE_CXXFLAGS += -fopenmp
```

Comment:
Here as well. openmp flag can be added on line 18

Reply:
Done